Buildings/Applications/DHC/CentralPlants/Cooling/Controls/ChillerStage.mo

```diff
@@ -0,0 +1,152 @@
+within Buildings.Applications.DHC.CentralPlants.Cooling.Controls;
+model ChillerStage "Stage controller for chillers"
+  extends Modelica.Blocks.Icons.Block;
+
+  parameter Modelica.SIunits.Time tWai "Waiting time";
+  parameter Modelica.SIunits.Power QEva_nominal
+    "Nominal cooling capaciaty (negative)";
+  parameter Modelica.SIunits.Power  criPoiLoa = 0.55*QEva_nominal
+    "Critical point of cooling load for switching one chiller on or off";
+  parameter Modelica.SIunits.Power  dQ = 0.25*QEva_nominal
+    "Deadband for critical point of cooling load";
+
+  Modelica.Blocks.Interfaces.RealInput QLoa(unit="W",max=0)
+    "Total cooling load, negative"
+    annotation (Placement(transformation(extent={{-140,-60},{-100,-20}})));
+  Modelica.Blocks.Interfaces.BooleanInput on "On signal of the chillers"
```

Comment:
Move this interface connector `on` up to be above input connector `QLoa`, so the order becomes consistent as the one show in Diagram graphical layer.

Also, does this signal mean to enable chiller, or enable plant (as in some plant, it might have economizer)? For the plant that have both chiller and waterside economizer (WSE), enabling plant would firstly enable WSE. Later stage would then enable chiller. Better clarify this in the comment.
```
True: enable chiller
```